default-config.yml

```diff
@@ -13,44 +13,31 @@ receivers:
     protocols:
       grpc:
       http:
-
   opencensus:
-
-  # Collect own metrics
-  prometheus:
-    config:
-      scrape_configs:
-        - job_name: "otel-collector"
-          scrape_interval: 10s
-          static_configs:
-            - targets: ["0.0.0.0:8888"]
-
   jaeger:
     protocols:
       grpc:
       thrift_binary:
       thrift_compact:
       thrift_http:
-
   zipkin:
 
 processors:
   batch:
 
 exporters:
-  logging:
-    logLevel: debug
+  kafka:
+    protocol_version: 2.0.0
+    brokers:
+      - kafka-zookeeper:9092
```

Comment:
Should be good to have `bootstrap:9092` and adding bootstrap as another alias in docker-compose?